### PR TITLE
robotont fake rviz

### PR DESCRIPTION
**Pull request by iharsuvorau**
That's the fake odom with rzviz stuff.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,202 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(robotont_support)
+
+## Compile as C++11, supported in ROS Kinetic and newer
+# add_compile_options(-std=c++11)
```
Comment (RValner) [inline CMakeLists.txt]: Uncomment the std c++11 compile options.

File: CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+
+## Find catkin macros and libraries
+## if COMPONENTS list like find_package(catkin REQUIRED COMPONENTS xyz)
+## is used, also find other catkin packages
+find_package(catkin REQUIRED)
+
+## System dependencies are found with CMake's conventions
```
Comment (RValner) [inline CMakeLists.txt]: Remove all the unnecessary commented junk. But don't remove "catkin_package(..." while you are doing that.

File: package.xml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  <maintainer email="parallels@todo.todo">parallels</maintainer>
+
+
+  <!-- One license tag required, multiple allowed, one license per tag -->
+  <!-- Commonly used license strings: -->
+  <!--   BSD, MIT, Boost Software License, GPLv2, GPLv3, LGPLv2.1, LGPLv3 -->
+  <license>TODO</license>
+
+
+  <!-- Url tags are optional, but multiple are allowed, one per tag -->
```
Comment (RValner) [inline package.xml]: Same here, remove all unnecessary comments.

**Review by RValner — CHANGES_REQUESTED** [review]
Refer to the comments.

**Comment by iharsuvorau** [comment]
@RValner Check this e41e653.

**Comment by Veix123** [comment]
Fake driver moved under https://github.com/robotont/robotont_driver repo.

**Pull request closed without merging**